### ISSUE TITLE
ijs.await - doesn't seem to wait for the response

### ISSUE THREAD

**Issue by paulroth3d** [paulroth3d/jupyter-ijavascript-utils#8]
ijs.await is not only supposed to allow for await calls
but it should also halt execution of the following cells until the await is complete.

ex:

```
utils.ijs.await(async($$, console) => {
   cars = await conn.queryCars();
});
```

so the following cell that uses cars can safely assume that `cars` should be established at this point.

But it seems like it isn't waiting for the response.

Look into it further...

**Comment by paulroth3d** [comment]
Fixed in build 1.9.1
Seems it wasn't actually waiting.

Closing when I hear confirmation it is resolved.

**Comment by paulroth3d** [comment]
Confirmed working - Thanks Rob!
